test/modules/map/components/olMap/handler/measure/MeasurementOverlay.test.js

```diff
@@ -10,7 +12,10 @@ describe('MeasurementOverlay', () => {
 
 	beforeEach(async () => {
 		TestUtils.setupStoreAndDi({});			
-
+		$injector.registerSingleton('ConfigService', {
+			getValue: () => { }
+		});
+		$injector.registerSingleton('UnitsService', new UnitsService());
```

Comment:
We only  use mocked services  in tests, because we want to focus just on the logic of the code under test and exclude foreign classes and their complexity